src/views/AccountPage.vue

```diff
@@ -171,6 +181,13 @@ export default defineComponent({
     },
     cancelEditing () {
       this.editingTitle = false
+    },
+    incomingOutgoingFlowClass (operation: Operation): string {
```

Comment:
I wasn't sure of the role of the function by seeing the name in the template. 
I would suggest something like... getClassDependingOnAmount() ?

Reply:
changed